### PR TITLE
Fixed Jenkins build

### PR DESCRIPTION
**Pull request by PradyumnaNagendra**

### CODE REVIEW

File: pipelines/build/synctool/Jenkinsfile [hunk 1]
```diff
@@ -11,17 +11,9 @@ node() {
             stage('Checkout') {
                 cleanWs()
                 def scmVars = checkout scm
```
Comment (keshavprasadms) [inline pipelines/build/synctool/Jenkinsfile:13]: Please remove this line

Comment (keshavprasadms) [inline pipelines/build/synctool/Jenkinsfile:13]: Remove

File: pipelines/build/yarn/Jenkinsfile [hunk 2]
```diff
@@ -11,18 +11,9 @@ node() {
             stage('Checkout') {
                 cleanWs()
                 def scmVars = checkout scm
```
Comment (keshavprasadms) [inline pipelines/build/yarn/Jenkinsfile:13]: Remove

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Check comments

File: pipelines/build/yarn/Jenkinsfile (hunk 2 above)
Comment (PradyumnaNagendra) [reply pipelines/build/yarn/Jenkinsfile]: done

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**